include/bzfsAPI.h

```diff
@@ -990,6 +993,17 @@ class BZF_API bz_PlayerUpdateEventData_V1 : public bz_EventData
  double stateTime;
 };
 
+class BZF_API bz_ComputeHandiCap_V1 : public bz_EventData
+{
+public:
+	bz_ComputeHandiCap_V1() : bz_EventData(bz_eComputeHandicapEvent)
+		, playerID(-1), desiredHandicap(0)
+	{}
+
+	int playerID;
+	int desiredHandicap;
```

Comment:
Indentation for this, can it be spaces instead of tabs to follow the rest of the file